lib/src/layer.dart

```diff
@@ -67,9 +70,54 @@ class Layer {
         tileIndex += 4;
 
         // Read out the flags
-        var flipped_horizontally = (globalTileId & FLIPPED_HORIZONTALLY_FLAG);
-        var flipped_vertically = (globalTileId & FLIPPED_VERTICALLY_FLAG);
-        var flipped_diagonally = (globalTileId & FLIPPED_DIAGONALLY_FLAG);
+        bool flippedHorizontally = (globalTileId & FLIPPED_HORIZONTALLY_FLAG) == FLIPPED_HORIZONTALLY_FLAG;
+        bool flippedVertically = (globalTileId & FLIPPED_VERTICALLY_FLAG) == FLIPPED_VERTICALLY_FLAG;
+        bool flippedDiagonally = (globalTileId & FLIPPED_DIAGONALLY_FLAG) == FLIPPED_DIAGONALLY_FLAG;
+
+        //Translating the flags into rotation
+        if(
```

Comment:
@Arne1303 after a lot of discussion we decided that this code should not be on tiled.dart since this lib is supposed to be kind of a mirror of how the tiled works.

So what tiled.dart must do is to expose these 3 booleans: `flippedHorizontally`, `flippedVertically` and `flippedDiagonally`, instead of the rotation. And the renderer (on our case Flame) should do this logic.

Let us know if you are still interested in continue with this PR, otherwise we can finish it :)

Reply:
I'm still interested in it :>